sky/serve/constants.py

```diff
@@ -0,0 +1,19 @@
+"""Constants used for SkyServe."""
+
+CONTROLLER_PREFIX = 'sky-serve-controller-'
+
+CONTROLLER_TEMPLATE = 'sky-serve-controller.yaml.j2'
+
+SERVE_PREFIX = '~/.sky/serve'
+
+CONTROLLER_PORT = 31001
+CONTROLLER_SYNC_INTERVAL = 20
```

Comment:
Add basic comments to exported constants? E.g., looking at CONTROLLER_PREFIX, I wondered if it's used as VM name. And not sure what CONTROLLER_SYNC_INTERVAL is for.